src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_instantiate.yaml

```
@@ -0,0 +1,46 @@
+post:
+  summary: Instantiate a new package through a project link.
+  description: |
+    Instantiate a new package in the specified project based on the packages inherited
+    through a linked project (local). The new package will be a branch of the package
+    that lives in the linked project unless the `makeoriginolder` parameter is provided.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: makeoriginolder
+      schema:
+        type: string
+      description: |
+        Instead of creating a branch of the package, it will instantiate a copy
+        of the package and extend the `vrev` to ensure the copy represents a
+        newer version of the package.
```

```suggestion
    - in: query
      name: makeoriginolder
      schema:
        type: string
      description: |
        Set to `1` to instantiate a copy of the package, instead of creating a branch of the package,
        and extend the `vrev` to ensure the copy represents a newer version of the package.
      example: 1
```
The `makeoriginolder` behaves like a boolean parameter in the backend. We could add an `example: 1`, and also change a little the description. What do you think?

---

@eduardoj Right now it just checks for presence of the parameter https://github.com/openSUSE/open-build-service/blob/91173f6cf9fc1e96a3fd500300c58357cf93c013/src/api/app/controllers/source_controller.rb#L840

---

I see. We can leave it this way.